app/reports/mi-report/group-events-by-correlation-id.js

```diff
@@ -0,0 +1,17 @@
+const groupEventsByCorrelationId = (events) => {
+  return [...events.reduce((x, y) => {
+    // group by correlation key, so create key representing the combination
+    // exclude account code as past requests vary based on ledger
+    const key = y.partitionKey
+
+    // if key doesn't exist then first instance so create new group
```

Comment:
Do we need the comments here anymore?

Reply:
I've removed them, mostly because they talk about account codes and ledgers which is irrelevant.  Clearly I just stole this from somewhere else during the spike...